app.js

```diff
@@ -1,89 +1,62 @@
-
-const myPWD = process.env.DB_PWD || 'Glock%2334';
-const myHOST = process.env.DB_HOST || 'localhost';
-const myUSR = process.env.DB_USER || 'admin';
-const myDB = process.env.DB_NAME || 'UMockMe';
-const myPLATFORM = process.env.DB_PLATFORM || 'mysql';
-
 const express = require('express');
 
-const app = express();
-
 const morgan = require('morgan');
 const bodyParser = require('body-parser');
 
-const sequelize = require('./db/sequelize')
-//const {Sequelize, Model, DataTypes} = require('sequelize');
-
-
-
-const prospectRoutes = require('./api/routes/prospects');
```

Comment:
I removed these routes since we'll likely want to revisit all of them before turning them on.